jobs/upload-brew.Jenkinsfile

```diff
@@ -0,0 +1,128 @@
+import org.yaml.snakeyaml.Yaml;
+
+def pipeutils, pipecfg, libcloud, uploading, jenkins_agent_image_tag
+def src_config_url, src_config_ref, brew_principle, brew_profile
+node {
+    checkout scm
+    pipeutils = load("utils.groovy")
+    pipecfg = pipeutils.load_pipecfg()
+    libcloud = load("libcloud.groovy")
+
+    brew_principle =  pipecfg['brew']['principle']
+    brew_profile =  pipecfg['brew']['profile']
+}
+
+arches = [ 'aarch64', 'ppc64le', 'x86_64', 's390x']
+
+properties([
+    pipelineTriggers([]),
+    parameters([
+      choice(name: 'ARCH',
+             description: 'The target architecture',
+             choices: (arches),
+             trim: true),
+      string(name: 'VERSION',
+             description: 'Build version',
+             defaultValue: '',
+             trim: true),
+      choice(name: 'STREAM',
+             choices: pipeutils.get_streams_choices(pipecfg),
+             description: 'CoreOS stream to upload to brew'),
+    ]),
+    buildDiscarder(logRotator(
+        numToKeepStr: '100',
+        artifactNumToKeepStr: '100'
+    )),
+    durabilityHint('PERFORMANCE_OPTIMIZED')
+])
+
+def build_description = "[${params.VERSION}]"
+
+// Always use the latest
+cosa_img = "quay.io/coreos-assembler/coreos-assembler:latest"
```

Comment:
should probably use the same machanism for this as we do the other jobs: 

https://github.com/coreos/fedora-coreos-pipeline/blob/0c6fef51773fe60d9e18083b2c6159e74ec6e3e0/jobs/build.Jenkinsfile#L75-L77

It is true that we want to use the non stream specific COSA, so maybe we modify it in some way. Either way, should still be picking this up from the pipecfg.